lib/utils/utils.go

```diff
@@ -0,0 +1,28 @@
+package utils
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+)
+
+// MakeRequest makes an API request.
+// It returns a response object and an error object.
+func MakeRequest(payload interface{}, URL, method string) (*http.Response, error) {
```

Comment:
A small nitpick here:
I'ld order the arguments as `method`, `URL`, `payload`.

Reply:
sounds good to me.